roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -200,7 +206,7 @@ def __init__(self, subject_string):
         self.subjects = []
         for s in subject_string.split(', '):
             name, _, value = s.partition('=')
-            self.subjects.append((name, value))
+            self.subjects.append((name.encode(), value.encode()))
```

Comment:
It looks like cryptography suffers from the same `bytes` problem and is also not available on atomic host.